ddtrace/contrib/httpx/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""
+The httpx__ integration traces all HTTP requests made with the ``httpx``
+library.
+
+Enabling
+~~~~~~~~
+
+The ``httpx`` integration is enabled automatically when using
+:ref:`ddtrace-run<ddtracerun>` or :ref:`patch_all()<patch_all>`.
+
+Alternatively, use :ref:`patch()<patch>` to manually enable the integration::
+
+    from ddtrace import patch
+    patch(httpx=True)
+
+    # use httpx like usual
+
+
+Global Configuration
+~~~~~~~~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.httpx['service']
+
+   The default service name for ``httpx`` requests. This value will
+   be overridden by an instance override or if the ``split_by_domain`` setting is
+   enabled.
+
+   This option can also be set with the ``DD_HTTPX_SERVICE`` environment
+   variable.
+
+   Default: ``None`` (inherit from ``DD_SERVICE``)
+
+
+.. py:data:: ddtrace.config.httpx['distributed_tracing']
+
+   Whether or not to inject distributed tracing headers into requests.
```

Comment:
What are the benefits of enabling distributed tracing with httpx? What functionality do the distributed tracing headers add?

Reply:
The ability to propagate the tracing context to any downstream services so the spans can be connected in the same trace